code/modules/integrated_electronics/subtypes/output.dm

```diff
@@ -204,6 +204,20 @@
 		)
 	spawn_flags = IC_SPAWN_RESEARCH
 
+/obj/item/integrated_circuit/output/sound/vox
+	name = "ai vox sound circuit"
+	desc = "Takes a sound name as an input, and will play said sound when pulsed. This circuit is often found in AI announcement systems."
+	spawn_flags = IC_SPAWN_RESEARCH
+
+/obj/item/integrated_circuit/output/sound/vox/Initialize()
+	sounds = GLOB.vox_sounds
+	.= ..()
+	extended_desc = list()
+	extended_desc += "The first input pin determines which sound is used. It uses the AI Vox Broadcast word list"
+	extended_desc += ". So either experiment to find words that work, or ask the AI to help in figuring them out"
+	extended_desc += ". The second pin determines the volume of sound that is played"
+	extended_desc += ", and the third determines if the frequency of the sound will vary with each activation."
+	extended_desc = jointext(extended_desc, null)
```

Comment:
Just type the whole thing out in one string, this is just wasted CPU/memory

Reply:
I'll do so, but that's how the other sound output circuits are constructed as well so maybe that should be addressed.